R/layers.R

```diff
@@ -206,8 +206,17 @@ write_file_attachments <- function(file_attachments, output_path) {
   output_path <- normalizePath(output_path, mustWork = TRUE)
 
   mapply(function(dest, src) {
-    if (fs::is_dir(src)) {
-      fs::dir_copy(src, file.path(output_path, dest), overwrite = TRUE)
+    if (dir.exists(src)) {
```

Comment:
Can we also add a comment about how this used to be `fs::dir_copy()`, but it's been changed for performance (and link to your comment)?